src/chainstate/burn/operations/mod.rs

```diff
@@ -329,6 +335,33 @@ pub struct PegInOp {
     pub burn_header_hash: BurnchainHeaderHash, // hash of the burn chain block header
 }
 
+#[derive(Debug, PartialEq, Clone, Eq, Serialize, Deserialize)]
+pub struct PegOutRequestOp {
+    pub amount: u64,                 // sBTC amount to peg out, in satoshis
+    pub recipient: PoxAddress,       // Address to receive the BTC when the request is fulfilled
+    pub signature: MessageSignature, // Signature from sBTC owner's private key(s)
+
+    // common to all transactions
```

Comment:
Please also store the extra bytes that aren't used for decoding, as a `memo: Vec<u8>`.

Reply:
Done :heavy_check_mark: